lib/bolt/outputter/json.rb

```diff
@@ -44,7 +44,11 @@ def print_summary(results, elapsed_time)
       end
 
       def print_plan(result)
-        @stream.puts result.to_json
+        if result_hash = unwrap_execution_result(result)
+          @stream.puts result_hash.to_json
+        else
+          @stream.puts result.to_json
```

Comment:
Won't this blow up for some objects? I think we need to fall back to "\"#{result.to_string}\"" or { "_output": "\"#{result.to_string}\""}

Reply:
It's what was originally there. It might blow up depending on what's returned.

Reply:
Amazingly it works with a plain string, as shown by one of the spec tests.

Reply:
Yeah I think it's blowing up now

Reply:
What was it failing on? I don't think it was related to the failing tests on the PR.